src/JestExt.ts

```diff
@@ -545,4 +552,47 @@ export class JestExt {
     const filePath = document.fileName
     this.testResultProvider.removeCachedResults(filePath)
   }
+
+  removeCachedDecorationTypes(document: vscode.TextDocument) {
+    if (!document || !document.fileName) {
+      return
+    }
+
+    delete this.failingAssertionDecorators[document.fileName]
+  }

```

Comment:
nice last week I almost made a PR for all this - https://github.com/jest-community/vscode-jest/tree/turn_off_stuff

Reply:
well, the removal anyway

Reply:
Nice. Assign me to the review if you get **that** annoyed with them again. :wink: